tests/h/search/index_test.py

```diff
@@ -310,6 +316,75 @@ def test_annotation_is_marked_deleted(self, es_client, factories, index, search)
         assert result.get('_source').get('deleted') is True
 
 
+class TestBatchIndexer(object):
+    def test_it_indexes_all_annotations(self, batch_indexer, es_client, factories):
+        annotations = factories.Annotation.create_batch(3)
+        ids = [a.id for a in annotations]
+
+        batch_indexer.index()
+
+        results = es_client.conn.search()['hits']['hits']
+        assert [ann.get("_id") for ann in results].sort() == ids.sort()
+
+    def test_it_indexes_specific_annotations(self, batch_indexer, es_client, factories):
+        annotations = factories.Annotation.create_batch(5)
+        ids = [a.id for a in annotations]
+        ids_to_index = ids[:3]
+        ids_not_to_index = ids[3:]
+
+        batch_indexer.index(ids_to_index)
+
+        results = es_client.conn.search()['hits']['hits']
+        assert [ann.get("_id") for ann in results].sort() == ids_to_index.sort()
+
+        for id in ids_not_to_index:
+            with pytest.raises(elasticsearch1.exceptions.NotFoundError):
+                es_client.conn.get(index=es_client.index,
+                                   doc_type="annotation",
+                                   id=id)
+
+    def test_it_logs_indexing_status(self, caplog, batch_indexer, factories):
+        annotations = factories.Annotation.create_batch(10)
+        ids = [a.id for a in annotations]
+
+        with caplog.at_level(logging.INFO):
+            batch_indexer.index(ids, 3)
+
+        for record in caplog.records:
```

Comment:
This test still passes if you comment out the `log.info` statement in `h/search/index.py` because in that case `caplog.records` is empty.